src/Microsoft.AspNetCore.Hosting.Abstractions/exceptions.net45.json

```diff
@@ -35,5 +35,40 @@
     "NewTypeId": "public interface Microsoft.AspNetCore.Hosting.IWebHost : System.IDisposable",
     "NewMemberId": "System.Threading.Tasks.Task StopAsync(System.Threading.CancellationToken cancellationToken)",
     "Kind": "Addition"
+  },
+  {
+    "OldTypeId": "public interface Microsoft.AspNetCore.Hosting.IApplicationLifetime",
+    "NewTypeId": "public interface Microsoft.AspNetCore.Hosting.IApplicationLifetime : Microsoft.Extensions.Hosting.IHostLifetime",
+    "Kind": "Modification"
+  },
+  {
+    "OldTypeId": "public interface Microsoft.AspNetCore.Hosting.IHostingEnvironment",
+    "NewTypeId": "public interface Microsoft.AspNetCore.Hosting.IHostingEnvironment : Microsoft.Extensions.Hosting.IApplicationEnvironment",
+    "Kind": "Modification"
+  },
+  {
+    "OldTypeId": "public interface Microsoft.AspNetCore.Hosting.IApplicationLifetime",
+    "OldMemberId": "System.Void StopApplication()",
+    "Kind": "Removal"
```

Comment:
This tool is being updated to just have add/remove, and no "modified". cc @smitpatel as FYI.